src/estimation/src/BerdyHelper.cpp

```diff
@@ -1526,6 +1589,20 @@ bool BerdyHelper::updateKinematicsFromFloatingBase(const JointPosDoubleArray& jo
     m_jointPos = jointPos;
     m_jointVel = jointVel;
 
+    // Update kinDyn computation
+    iDynTree::KinDynComputations kinDynComputations;
+    kinDynComputations.loadRobotModel(m_model);
+    kinDynComputations.setRobotState(m_jointPos,
+                                     m_jointVel,
+                                     m_gravity);
+
+    // TODO: Update base_H_m_links transformations
+     for(LinkIndex idx = 0; idx < static_cast<LinkIndex>(m_model.getNrOfLinks()); idx++)
+     {
+         base_H_m_links.at(idx) = kinDynComputations.getRelativeTransform(idx, m_model.getLinkIndex(m_options.baseLink));
+         //std::cout << "Berdy helper : updated transform for the link " << m_model.getLinkName(idx) << " with the transform : " << base_H_m_links.at(idx).toString().c_str() << std::endl;
```

Comment:
This print if not significant can be removed. Otherwise consider using `iDynTree::reportInfo()`, so that we can easily identify iDynTree related prints on the console.

Reply:
Removed the debug code in https://github.com/robotology/idyntree/pull/571/commits/f57ab3c061b91f83c5351d671132cc922ca75556